mathics/builtin/natlang.py

```diff
@@ -119,6 +119,10 @@ def generator(doc):
         'Adjective': [nltk.corpus.wordnet.ADJ, nltk.corpus.wordnet.ADJ_SAT],
         'Adverb': [nltk.corpus.wordnet.ADV],
     }
+except LookupError:
+    print("warning: nltk dictionary not installed\n")    
```

Comment:
To be more clear: What I think is going on is that the wordnet corpus is missing? Fixing the `LookupError` in this way will just move the error to a `KeyError` in the natural language functions. This is better but doesn't properly resolve the issue.